lib/conjur_client.rb

```diff
@@ -17,22 +17,12 @@ def readonly_api
       ConjurClient.new.api(ConjurClient.application_conjur_url)
     end
 
-    def v4_host_factory_id
-      if policy == "root"
-        "#{account}:host_factory:apps"
-      else
-        "#{account}:host_factory:#{policy}/#{policy}-apps"
-      end
-    end
-
     def v5?
       version == 5
     end
 
     def version
       case ENV['CONJUR_VERSION']
-      when "4"
-        4
```

Comment:
Maybe it's worth adding a specific error here, like
`raise 'Conjur Enterprise v4 is no longer supported. Please use a service broker v1.1.4 or earlier.'`

Reply:
then in theory that will show in the health check.

Reply:
Good point - I added that error message in the `4` clause